README.md

```diff
@@ -105,7 +105,7 @@ This will bring up etcd listening on port 2379 for client communication and on p
 Next, let's set a single key, and then retrieve it:
 
 ```
-etcdctl put mykey "this is awesome"
+etcdctl set mykey "this is awesome"
```

Comment:
`put` is correct. You can read more https://github.com/etcd-io/etcd/tree/master/etcdctl  (for v2 https://github.com/etcd-io/etcd/blob/master/etcdctl/READMEv2.md ) Unless I'm missing anything, you should close the PR. Thanks!